packages/kerosene-test/src/jest/index.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Shorthand for `jest.Mock<ReturnType<T>, Parameters<T>>;`
+ */
+export type JestMock<T extends (...args: any[]) => any> = jest.Mock<
```

Comment:
if I want to strictly-type my args, would the `any` here make me lose them?

Reply:
This doesn't lose type information, it just specifies that `T` **should be a function** which takes any arguments and returns any value. `T` will retain the correct type information when passed into `Parameters<T>` and `ReturnType<T>`